CHANGELOG.md

```diff
@@ -123,7 +123,7 @@ New modules
 
 * `Data.Sign.Show` to show a sign
 
-Additions to existing modules
+* `Algebra.Properties.Semigroup` adding consequences for associtvity for semigroups
```

Comment:
Typo
```suggestion
* `Algebra.Properties.Semigroup` adding consequences for associativity for semigroups
```

Reply:
And, in case it wasn't clear before: you *do* need to list, explicitly, every single lemma as an addition to that module (follow the model of the entries below), so this line should also move downwards to a separate block between `Algebra.Properties.Monoid.Divisibility` and `Algebra.Properties.Semigroup.Divisibility`...
... AND you've somehow deleted the line
`Additions to existing modules` at L126... which you need to restore, sigh.